pkg/check/smoke/smoke.go

```diff
@@ -3,136 +3,132 @@ package smoke
 import (
 	"bytes"
 	"context"
+	"errors"
 	"fmt"
-	"math/rand"
 	"time"
 
+	"github.com/ethersphere/beekeeper/pkg/bee"
 	"github.com/ethersphere/beekeeper/pkg/bee/api"
 	"github.com/ethersphere/beekeeper/pkg/beekeeper"
 	"github.com/ethersphere/beekeeper/pkg/orchestration"
 	"github.com/ethersphere/beekeeper/pkg/random"
+	"github.com/google/uuid"
 )
 
-// TODO: remove need for node group, use whole cluster instead
-
 // Options represents smoke test options
 type Options struct {
-	Bytes           int // how many bytes to upload each time
-	NodeGroup       string
-	Runs            int // how many runs to do
-	Seed            int64
-	Timeout         time.Duration
-	UploadNodeCount int
+	ContentSize   int64
+	Iterations    int
+	RndSeed       int64
+	GasPrice      string
+	PostageAmount int64
+	PostageDepth  uint64
+	PostageLabel  string
+	SyncTimeout   time.Duration
 }
 
 // NewDefaultOptions returns new default options
 func NewDefaultOptions() Options {
 	return Options{
-		Bytes:           0,
-		NodeGroup:       "bee",
-		Runs:            1,
-		Seed:            0,
-		Timeout:         1 * time.Second,
-		UploadNodeCount: 1,
+		ContentSize:   5 * 1024 << 10,
+		Iterations:    1,
+		RndSeed:       0,
+		GasPrice:      "",
+		PostageAmount: 1000,
+		PostageDepth:  16,
+		PostageLabel:  "test-label",
+		SyncTimeout:   time.Second,
```

Comment:
this is too short I believe, let's make this like 1m

Reply:
Done.